src/components/TheFooter.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <div class="footer-combined">
+        <div class="upper">
+            <div class="left">
+                <router-link to="Privacy"></router-link>
+                <a href="./Privacy.vue">Privacy</a>
+                <a href="./Terms.vue">Terms</a>
+            </div>
+            <div class="right">
+                <img src="@/assets/facebook.svg" alt="">                
+            </div>
+        </div>
+        <div class="lower"> 
+            
+            © 2022 Daily Stocks, all rights reserved
+        </div>
+        
+
+    </div>
+</template>
+
+<style scoped>
+
+    .fb-icon{
+        height: 64px;
+        background: rebeccapurple;
+        border: 2px;
+    }
```

Comment:
```suggestion
    .fb-icon {
        height: 64px;
        background: rebeccapurple;
        border: 2px;
    }
```
give space before `{`

Also, it is good to use indent using space "2". So the code turns up less wide.

Apply this wherever applicable